tools/kbump/pkg/kbump/kbump.go

```diff
@@ -0,0 +1,82 @@
+package kbump
+
+import (
+	"errors"
+	"sigs.k8s.io/yaml"
+
+	"github.com/aws/karpenter/pkg/apis/v1alpha1"
+	nodeclassutil "github.com/aws/karpenter/pkg/utils/nodeclass"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"github.com/aws/karpenter-core/pkg/apis/v1alpha5"
+	nodepoolutil "github.com/aws/karpenter-core/pkg/utils/nodepool"
+)
+
+type Parameters struct {
+	Role *string
+}
+
+func Process(inputYAML []byte, params *Parameters) ([]byte, error) {
+	var input metav1.TypeMeta
+	err := yaml.Unmarshal(inputYAML, &input)
+	if err != nil {
+		return nil, err
+	}
+
+	switch input.Kind {
+		case "Provisioner":
+			return processProvisioner(inputYAML)
+		case "AWSNodeTemplate":
+			return processNodeTemplate(inputYAML, params)
+		default:
+			return nil, errors.New("Unknown Kind")
```

Comment:
```suggestion
			return nil, fmt.Errorf("Unknown Kind")
```

Do you think this error should return more info to the user? Maybe some detail on which kinds we support on top of the fact that the kind that was passed through was unknown to us?